src/mscorlib/shared/System/IO/Win32Marshal.cs

```diff
@@ -65,8 +65,11 @@ internal static Exception GetExceptionForWin32Error(int errorCode, string path)
                 case Interop.Errors.ERROR_FILENAME_EXCED_RANGE:
                     return new PathTooLongException(SR.Format(SR.IO_PathTooLong_Path, path));
 
+                case Interop.Errors.ERROR_INVALID_DRIVE:
```

Comment:
This was missing in the __Error copy and it's possible this could cause some exceptions to now be DriveNotFoundException that were previously IOException. I figure that's OK as it's a derived type.

Reply:
There is exact same core in CoreFX as well. You may want to fix it there as well.

Reply:
will do